CBLClient/Apps/CBLTestServer-Java-WS/src/main/java/com/couchbase/javaws/TestServerWS.java

```diff
@@ -0,0 +1,116 @@
+package com.couchbase.javaws;
+
+import com.couchbase.mobiletestkit.javacommon.*;
+import com.couchbase.mobiletestkit.javacommon.util.Log;
+import com.couchbase.lite.CouchbaseLite;
+import com.couchbase.lite.Database;
+import com.couchbase.lite.LogDomain;
+import com.couchbase.lite.LogLevel;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.EnumSet;
+import java.util.Map;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(name = "TestServerWS", urlPatterns = {"/"}, loadOnStartup = 1)
+public class TestServerWS extends HttpServlet {
+    public static final String TAG = "WS-REQUEST";
+    public static final Memory memory = new Memory();
+
+    @Override
+    public void init() throws ServletException {
+        CouchbaseLite.init();
+        Database.log.getConsole().setLevel(LogLevel.DEBUG);
+        Database.log.getConsole().setDomains(EnumSet.of(LogDomain.ALL));
+
+        Log.init(new TestServerLogger());
+    }
+
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
```

Comment:
It might be a good idea to add doGet() to print out something for quick checking if the server is up or not by using browser.

Reply:
added doGet()